javascript/script.js

```diff
@@ -0,0 +1,10 @@
+// .script
+var list = document.getElementByID('list'),
+	add = document.getElementByID('addElem');
```

Comment:
nie deklaruj zmiennych po przecinku

jeśli konsola mówi, że nie ma takiej funkcji jak `getElementByID` to znaczy, że nie ma :) jest `getElementById`